src/app/helptext/services/components/service-s3.ts

```diff
@@ -47,5 +47,12 @@ mode_options : [
 certificate_placeholder : T('Certificate'),
 certificate_tooltip : T('Add an <a href="--docurl--/system.html#certificates"\
  target="_blank">SSL certificate</a> to be used for\
- secure S3 connections.')
+ secure S3 connections.'),
+
+path_warning_title : T('Warning'),
```

Comment:
Rough suggested rewrite:

'Selecting a dataset to use with Minio <b>removes all existing permissions for the dataset and any nested directories!</b> Permissions are reset to <code>minio:minio</code>. To avoid dataset permissions conflicts, please create a separate, dedicated dataset for Minio.'